browser/src/canvas/CanvasSectionContainer.ts

```diff
@@ -577,9 +577,34 @@ class CanvasSectionContainer {
 		while (layoutingService.runTheTopTask());
 	}
 
-	private resizeCanvas() {
+	private checkCanvasSizeAfterDisplayChanges(): boolean {
```

Comment:
I'm guilty of this a lot myself, but I don't think this function name helps understand what the return value means. What about something like `isCanvasSizeValidAfterDisplayChange`?